installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -0,0 +1,124 @@
+:_content-type: ASSEMBLY
+[id="installing-mirroring-disconnected"]
+= Mirroring images for a disconnected installation using the oc-mirror plug-in
+include::modules/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+////
+General to do list:
+* Confirm that no new content was added since I did the reorg of the disconnected install assemblies (up to date as of 2/13)
+* Add info about mirroring to a namespace
+* Test and get some actual example output
+* Use cases for changing image set config between runs
+////
+
+You can use the procedures in this section to ensure your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
+
+You can use the oc-mirror OpenShift CLI (`oc`) plug-in to mirror images to a mirror registry in your fully or partially disconnected environments.
+
+:FeatureName: Mirroring images for disconnected environments using the oc-mirror plug-in
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The following steps list the high-level workflow for using the oc-mirror plug-in to mirror images to a mirror registry:
+
+. Create an image set configuration file.
+. Mirror the image set to the mirror registry.
+. Installing the `ImageContentSourcePolicy` and `CatalogSource` resources into the cluster.
+. Repeat these steps to update your mirror registry as necessary.
+
+// About the oc-mirror plug-in
+include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+// TODO: Check the content of this module to make sure it's accurate for oc-mirror
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For information on viewing the CRI-O logs to view the image source, see xref:../../installing/validating-an-installation.adoc#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+// TODO: Copied this prereq section + about mirror registry from the orig section. Review/update as necessary
+[id="prerequisites_installing-mirroring-disconnected"]
+== Prerequisites
```

Comment:
@afflom @jpower432 I pulled in this list of prerequisites (and which supported registry list) from the current docs we have. Can you please review and let me know if it looks correct for our purposes?

The only change I think I made was adding that oc-mirror requires Quay 3.6 or later.

Preview here: https://deploy-preview-40474--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#prerequisites_installing-mirroring-disconnected

Reply:
Thanks @bergerhoffer, I would say the supported registry list for tech preview would be Quay (3.6 or later) and Docker Registry. @afflom Do you agree? Would it be worth it to verify the others right now?

Reply:
I agree with CNCF Distribution and Quay 3.6+. We've also tested go-containerregistry pretty extensively with oc-mirror without issue. Not sure if thats something that we want to advertise.

Reply:
@jpower432 @afflom So can you sync up and confirm which ones we want to list? I see consensus on Quay 3.6+, but I'm not sure if Docker Registry and CNCF are the same or different, etc.

Reply:
Docker registry was donated to CNCF. I think the names are interchangeable. Docker registry and quay 3.6+ are definitely tested. We need to test the others listed.  We dont feel comfortable mentioning them as compatible yet. We do know that the 4.10 release of oc-mirror does not work with Harbor. Can we tag artifactory and nexus as untested instead of unsupported/omitted so that we dont discourage people from testing them?